website/content/en/status/report-2025-04-2025-06/freebsd-foundation.adoc

```diff
@@ -0,0 +1,83 @@
+=== FreeBSD Foundation
+
+Links: +
+link:https://freebsdfoundation.org/[FreeBSD Foundation] URL: link:https://freebsdfoundation.org/[] +
+link:https://freebsdfoundation.org/blog/technology-roadmap/[Technology Roadmap] URL: link:https://freebsdfoundation.org/blog/technology-roadmap/[] +
+link:https://freebsdfoundation.org/donate/[Donate] URL: link:https://freebsdfoundation.org/donate/[] +
+link:https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/[Foundation Partnership Program] URL: link:https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/[] +
+link:https://freebsdfoundation.org/journal/[FreeBSD Journal] URL: link:https://freebsdfoundation.org/journal/[] +
+link:https://freebsdfoundation.org/our-work/events/[Foundation Events] URL: link:https://freebsdfoundation.org/our-work/events/[]
+
+Contact: Deb Goodkin <deb@FreeBSDFoundation.org>
+
+The FreeBSD Foundation is a 501(c)(3) non-profit dedicated to advancing FreeBSD through both technical and non-technical support.
+Funded entirely by donations, the Foundation supports software development, infrastructure, security, and collaboration efforts; organizes events and developer summits; provides educational resources; and represents the FreeBSD Project in legal matters.
+
+Here are some of the ways we supported FreeBSD in the second quarter of 2025.
+
+==== Advocacy
+
+Advocacy work in the 2nd quarter of 2025 included hosting events, launching a new series of video guides and bringing on a new Marketing Coordinator.
+Florine Kamdem brings social media, branding, and IT skills.
+She uses storytelling to craft digital campaigns that spark interest and build connection within the community.
+Read more about link:https://freebsdfoundation.org/about-us/our-team/[Florine], and check out just a few of the ways the Foundation helped advocate for FreeBSD in Q2 of 2025:
+
+* Held the link:https://wiki.freebsd.org/DevSummit/202506[June 2025 FreeBSD Developer Summit] June 11-12, 2025, co-located with link:https://www.bsdcan.org/2025[BSDCan 2025].
+  Videos of the all day stream are available on the Project's https://www.youtube.com/c/FreeBSDProject[YouTube Channel], and videos of the individual talks will be available in the coming weeks.
+* Finalized our Silver Sponsorship of https://2025.eurobsdcon.org/[EuroBSDcon 2025], held in Zagreb, Croatia; September 25-28, 2025.
+  link:https://forms.gle/3gagxxLtxprZBJoz7[Travel Grants] are now available.
```

Comment:
I would rather not use link shorteners. Using real urls looks safer to me (are short urls reused over time? is it possible for an attacker to hijack them? I do not know) and the link will not appear in the text anyway on the web rendered version.

```suggestion
  link:https://docs.google.com/forms/d/e/1FAIpQLSfj82hLQ8-uKrr1B6MmgnhzZj7pzMaTjA708hszSLHwDn7msA/viewform[Travel Grants] are now available.
```